packages/cli/package.json

```diff
@@ -94,6 +94,7 @@
     "@types/write-json-file": "2.2.1",
     "@types/yauzl-promise": "2.1.0",
     "@vercel-internals/types": "*",
+    "@vercel-internals/utils": "*",
```

Comment:
lets not merge until we verify the cli builds/installs/runs as expected (like lets do a manual test with the preview tarball)